src/Message/ImpersonateMessage.php

```diff
@@ -28,23 +28,21 @@ public function getTo()
 
     public function getCc()
     {
-        return null;
```

Comment:
I just don't agree with this rules (which is incompatible with proper return types btw)